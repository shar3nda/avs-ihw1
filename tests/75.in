7
-1 3 9 8 6 -2 8