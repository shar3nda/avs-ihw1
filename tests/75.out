-7 -8 -8 -3 -3 0 0 -9 2 10